docs/reference/search-connectors/es-connectors-azure-blob.md

```diff
@@ -163,7 +163,7 @@ If you’re running the connector service against a Dockerized version of Elasti
 
 ```yaml
 # When connecting to your cloud deployment you should edit the host value
-elasticsearch.host: http://host.docker.internal:9200
+elasticsearch.host: <DOCKER_HOST_URL>
```

Comment:
I've lost genuine connection details here as a user tho, 🤔, same applies to all these connectors 

I'm not sure I understand the potential attack vector here

Reply:
Yup, I agree. I was overzealous. :-)

I'm replacing them all back to the original...